lib/src/components/NxThreatCounter/NxThreatCounter.tsx

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import classnames from 'classnames';
+
+import { Props, propTypes } from './types';
+export { Props } from './types';
+
+import './NxThreatCounter.scss';
+
+export default function NxThreatCounter({
+  criticalCount, severeCount, moderateCount, lowCount, layout, className, ...attrs }: Props) {
+
+  const layoutClasses = classnames('nx-threat-counter-container', className, {
+    [`nx-threat-counter-container--${layout}`]: layout
+  });
+
+  return (
+    <div className={layoutClasses} {...attrs}>
+      <dl className="nx-threat-counter nx-threat-counter--critical">
```

Comment:
I intended that the container would be the `<dl>` and the counters would be `<div>`s

Reply:
> I intended that the container would be the `<dl>` and the counters would be `<div>`s

Ah, https://github.com/sonatype/sonatype-react-shared-components/pull/343/commits/e03e606a5c880e49c5b2be926c41c27df75c36de